support-figma/extended-layout-plugin/src/ui.html

```diff
@@ -937,8 +938,32 @@
     return td;
   }
 
-  function createNodeIdElement(node) {
+  function createNodeIdElement(node, isSingleNode) {
     const block = document.createElement('div');
+
+    if (!isSingleNode) {
+      const editBtn = document.createElement('span');
+      editBtn.style.width = '20px';
+      editBtn.style.height = '20px';
+      editBtn.style.padding = '0px';
+      editBtn.classList.add('icon-button');
+      const editIcon = document.createElement('span');
+      editIcon.classList.add('icon', 'icon--break');
+      editIcon.style.width = '16px';
+      editIcon.style.height = '16px';
+      editBtn.appendChild(editIcon);
+      editBtn.onclick = () => {
+        parent.postMessage({
+          pluginMessage: {
+            msg: 'localization-ungroup-node',
+            node: node["nodeId"],
+            stringResMap: Array.from(outputStringData)
+          }
+        }, '*');
+      };
+      block.append(editBtn);
```

Comment:
When I mouse over this button, it changes to a hand icon for 1-2 pixels on the perimeter of the button, then back to an arrow in the center. I think it's changing to the hand when it's over the 'icon-button' but not over the 'icon--break' icon. Can you fix this so it's either the arrow or the hand icon the whole time? I also think a short tooltip would help, even though there are instructions at the top.

Reply:
Done.